app/javascript/components/student/TracksList/List/Track.jsx

```diff
@@ -4,29 +4,27 @@ import { ProgressBar } from './Track/ProgressBar'
 
 export function Track({ track }) {
   return (
-    <tr>
-      <td>
-        <TrackIcon track={track} />
-      </td>
-      <td>{track.title}</td>
-      <td>{track.numConceptExercises}</td>
-      <td>{track.numPracticeExercises}</td>
-      <td>{track.webUrl}</td>
-      <td>{track.isNew.toString()}</td>
-      <td>{track.tags.join(', ')}</td>
-      <td>{track.isJoined.toString()}</td>
-      <td>{track.numCompletedConceptExercises}</td>
-      <td>{track.numCompletedPracticeExercises}</td>
-      <td>
-        {track.isJoined && (
-          <ProgressBar
-            numConceptExercises={track.numConceptExercises}
-            numPracticeExercises={track.numPracticeExercises}
-            numCompletedConceptExercises={track.numCompletedConceptExercises}
-            numCompletedPracticeExercises={track.numCompletedPracticeExercises}
-          />
-        )}
-      </td>
-    </tr>
+    <a className="c-track" href={track.webUrl}>
+      <TrackIcon track={track} />
+      <div className="info">
+        <h3 className="title">{track.title}</h3>
+        <ul className="counts">
+          <li>
+            {track.numCompletedConceptExercises}/{track.numConceptExercises}{' '}
+            concepts
+          </li>
+          <li>
+            {track.numCompletedPracticeExercises}/{track.numPracticeExercises}{' '}
+            exercises
+          </li>
+        </ul>
+        <ul className="tags">
+          {track.tags.slice(0, 3).map((tag) => {
+            return <li>{tag.split(':')[1]}</li>
+          })}
+        </ul>
+      </div>
+      <i>›</i>
```

Comment:
This will be a proper icon but that involves working out how to add the icon library via NPM so it's a job for a different PR.

Reply:
Aha, so _this_ is the weird character being displayed :)